server/test/article-search.spec.js

```diff
@@ -0,0 +1,35 @@
+import chai, { expect } from 'chai';
+import chaiHttp from 'chai-http';
+import app from '../../index';
+
+
+chai.use(chaiHttp);
+describe('GET /api/v1/article/search', () => {
+  it('should return 200 for search query successfully run', async () => {
+    const res = await chai.request(app)
+      .get('/api/v1/article/search?author=Mofe&tag=&title=The%20boy%20drank%20palm&20wine');
+    expect(res).to.have.status(200);
+    expect(res.body.data.rows).to.be.an('array');
```

Comment:
Good job testing for empty parameters and special characters. I think this test suite can be better improved by checking for the actual articles returned from the search. What do you think?

Reply:
Will do that

Reply:
I will implement this.

Reply:
Done